mne/channels/channels.py

```diff
@@ -440,18 +440,36 @@ def inst_has(attr):
 class InterpolationMixin(object):
     """Mixin class for Raw, Evoked, Epochs
     """
-    def interpolate_bads_eeg(self):
-        """Interpolate bad channels
 
-        Operates in place.
+    def interpolate_bads(self, reset_bads=True):
```

Comment:
Yeah, I agree it's a little bit nonstandard to have in-place operations return the object, but it's *really* convenient. So in this instance I don't mind changing the standard. Plus people don't have to make use of the `return self` if they don't want to, so in that sense it's the most flexible option, too.